gcp_variant_transforms/libs/processed_variant_test.py

```diff
@@ -488,3 +489,52 @@ def test_create_alt_bases_field_schema_descriptions(self):
     self.assertEqual(alt_field.description, 'Alternate base.')
     a2_fields = [field for field in schema.fields if field.name == 'A2']
     self.assertEqual(len(a2_fields), 1)
+
+  def test_create_alt_bases_field_schema_types(self):
+    _, header_fields = self._get_sample_variant_and_header_with_csq()
+    for hfi in header_fields.infos.values():
+      hfi['type'] = 'String'
+    header_fields.infos.update({
+        'CSQ_Allele_TYPE':
+        OrderedDict([('id', 'CSQ_Allele_TYPE'),
```

Comment:
Please use `vcf.parser._Info` for these. Some tests explicitly import `_Info as Info` to make it less awkward.
We'll replace this with VcfHeader class from nucleus eventually, so it would be nicer, so using the Info class makes it easier to find and replace all instances of PyVCF headers.